apps/docs/content/guides/database/postgres/roles.mdx

```diff
@@ -86,7 +86,7 @@ The default Postgres role. This has admin privileges.
 
 ### `anon`
 
-For "anonymous access." This is the role which the API (PostgREST) will use when a user _is not_ logged in.
+For unauthenticated, public access. This is the role which the API (PostgREST) will use when a user _is not_ logged in.
```

Comment:
maybe we can also add [below](https://github.com/supabase/supabase/pull/21814/files#diff-5556b95c3fe109bc51da5574574956f62c3b9303b87a13727ed098af1fa1b2ecR98) that the `authenticated` role is used by an anonymous user too?

Reply:
@kangmingtay, can you put that in your PR? I want to keep this one all pre-release of anonymous signin, and yours all post-release of anonymous signin, if that makes sense